content/articles/cloud-management/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: engineering-education
+title: Cloud Management
+description: Cloud management is the control of products and services running in the cloud. It caters for these products and services. In this article we will get to understand how to choose the best cloud platforms. Also, we'll discuss challenges of cloud management and ways to beat them.
+author: anyebe-blessing-ene
+date: 2021-08-06T00:00:00-10:30
+topics: {Cloud}
+excerpt_seperator: <!--more-->
+images: 
+---
+In this article, we will discuss how cloud management and understand what there is to know about it.
+<!--more-->
+
+### Table of Contents
+- Cloud Management
+- Cloud Management Platforms
+- Cloud Management Challenges
+- Cloud Management Benefits
+- Functionalites of a decent cloud manager
+
+### Prerequisities 
+Basic knowledge of cloud computing
+
+Let's dive right in by learning about the concept
+
+### Cloud Management
```

Comment:
Before you get into this topic, state about the current scenario.

Explain the current drawback. What's the need to manage? 

Don't just state facts. Walk the reader through the problem, and give a solution